### PR TITLE
Details modal/popup functionality build

### PR DESCRIPTION
**Pull request by MuhammadUsmanAslam**
- Created modal/popup for mobile and desktop versions.
- Dynamically populating data on each list item in the work section using JavaScript array and object.
- Dynamically populating each popup when the user clicks on the specific card in the list.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️ You have submitted a project, that the 😮 Lighthouse? 😮 linter check is not passing. Kindly fix the linters and submit your project again. 
Screenshot 2023-02-02 at 12 30 47 AM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MuhammadUsmanAslam** [comment]
Thank You for your response but I checked using these commands: npx stylelint "**/*.{css,scss}" and npx eslint . there was no error. How I can check for these errors can you please tell me?

**Comment by huseyinbiyik** [comment]
Hey @MuhammadUsmanAslam. You can check the [Technical Forum](https://questions.microverse.org/t/linter-lighthouse-error-code-1-setup-lighthouse/3974).
The issue might occur because you have many h1 elements on one page. h1 needs to be used once or you have details.html which has no h1 element. 
By the way, this is a one-pager, so I think, there should be only one HTML file. 👍

File: style.css [hunk 1]
```diff
@@ -395,6 +401,99 @@ textarea:focus {
...
+.detail-card-inner {
+  flex-direction: column;
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1em;
+  justify-content: center;
+  margin: 0.5em 2%;
+  padding: 1em;
+}
```
Comment (franclobo) [inline style.css]: - [x] Congrats 👏🏽 👏🏽 👏🏽 very good job at the first the modal page looks similar to the Figma design, just a couple of things to correct. When the modal window opens, it should overlap (z-index) the rest of the page and need to be fixed, kindly pay attention to the gray border around the pop-up window and use opacity to have a similar aspect to the Figma design. It is normal that the pop-up window high exceeded the screen size, so we can use `overflow-y: scroll` for a better user experience.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi team, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @MuhammadUsmanAslam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**